CHANGELOG.md

```diff
@@ -6,18 +6,23 @@
 - Don't include changes that are purely internal. The CHANGELOG should be a
   useful summary for people upgrading their application, not a replication
   of the commit log.
-  
-## 24.13.3
-
-* Add event tracking to header elements ([#2110](https://github.com/alphagov/govuk_publishing_components/pull/2110)) 
 
 ## Unreleased
+
 * Allow a custom link to be used on the header logo ([PR #2114](https://github.com/alphagov/govuk_publishing_components/pull/2114))
 
 ## Unreleased
 
 * Add custom tracking action to Brexit superbreadcrumbs ([PR #2118](https://github.com/alphagov/govuk_publishing_components/pull/2118))
 
+## Unreleased
+
+* Add superbreadcrumb to content tagged to a Brexit child taxon ([PR #2123](https://github.com/alphagov/govuk_publishing_components/pull/2123))
+
```

Comment:
Something went terribly wrong along the way as we ended up with 3 Unreleased headings 😬  – but I don't think it's been introduced in this PR.

Reply:
Yes but luckily someone has tidied that up on master, as it sorted itself out once I rebased :)